ios/FluentUI.Demo/FluentUI.Demo/Demos/BottomCommandingDemoController.swift

```diff
@@ -0,0 +1,258 @@
+            bottomCommandingController?.expandedListSections = expandedListSections
+        } else {
+            bottomCommandingController?.expandedListSections = []
+        }
+    }
+
+    private let modifiedCommandIndices: [Int] = [0, 2, 4]
+
+    @objc private func toggleHeroCommandOnOff() {
+        modifiedCommandIndices.forEach {
+            heroItems[$0].isOn.toggle()
+        }
+    }
+
+    @objc private func toggleHeroCommandEnabled() {
+        modifiedCommandIndices.forEach {
+            heroItems[$0].isEnabled.toggle()
+        }
+    }
+
+    @objc private func toggleListCommandEnabled() {
+        modifiedCommandIndices.forEach {
+            expandedListSections[0].items[$0].isEnabled.toggle()
+        }
+    }
+
+    @objc private func changeHeroCommandTitle() {
+        modifiedCommandIndices.forEach {
+            heroItems[$0].title = "Item " + String(Int.random(in: 6..<100))
+        }
+    }
+
+    @objc private func changeListCommandTitle() {
+        modifiedCommandIndices.forEach {
+            expandedListSections[0].items[$0].title = "Item " + String(Int.random(in: 6..<100))
+        }
+    }
+
+    @objc private func changeHeroCommandIcon() {
+        modifiedCommandIndices.forEach {
+            heroItems[$0].image = heroIconChanged ? homeImage : boldImage
+            heroItems[$0].selectedImage = heroIconChanged ? homeSelectedImage : boldImage
+        }
+        heroIconChanged.toggle()
+    }
+
+    @objc private func changeListCommandIcon() {
+        modifiedCommandIndices.forEach {
+            expandedListSections[0].items[$0].image = listIconChanged ? homeImage : boldImage
+            expandedListSections[0].items[$0].selectedImage = listIconChanged ? homeSelectedImage : boldImage
+        }
+        listIconChanged.toggle()
+    }
+
+    @objc private func incrementHeroCommands() {
+        let currentCount = bottomCommandingController?.heroItems.count ?? 0
+        if currentCount < 5 {
+            let newCount = currentCount + 1
+            bottomCommandingController?.heroItems = Array(heroItems[0..<newCount])
+        }
+    }
+
+    @objc private func decrementHeroCommands() {
+        let currentCount = bottomCommandingController?.heroItems.count ?? 0
+        if currentCount > 1 {
+            let newCount = currentCount - 1
+            bottomCommandingController?.heroItems = Array(heroItems[0..<newCount])
+        }
+    }
+
+    @objc private func commandAction(item: CommandingItem) {
+        if heroItems.contains(item) {
+            showMessage("Hero command tapped")
+        } else if expandedListSections.contains(where: { $0.items.contains(item) }) {
+            showMessage("Expanded list command tapped")
+        }
+    }
+
+    private func showMessage(_ message: String) {
+        let alert = UIAlertController(title: message, message: nil, preferredStyle: .alert)
+        present(alert, animated: true)
+
+        DispatchQueue.main.asyncAfter(deadline: .now() + 1) {
```

Comment:
do we need this?

Reply:
Yes because this isn't subclassing DemoController anymore. The messages were the only thing we were using from it, and the way it adds a UIStackView in viewDidLoad was breaking the view hierarchy. It looks like DemoController should be updated in the future, now that we're switching to table views for the config options.

Reply:
not really a fan of autodismiss. we can't just wait till user press ok or something to dismiss the alert? seems weird precedence and not very acessibiltiy friendly either.

Reply:
I agree with that, but for this kind of thing I dislike manual dismiss even more. We just want to indicate a command was triggered, ideally without taking away focus or anything like that. Maybe a NotificationView would be best for this sort of thing?

Reply:
I switched to dismissible alerts for now. In the future I'd like to use a lighter toast, but there was an issue where it was overlapping the collapsed sheet so I'd need to spend more time on that.